src/core/Akka.API.Tests/CoreAPISpec.ApproveCore.approved.txt

```diff
@@ -4902,6 +4901,12 @@ namespace Akka.Util
         public bool IsValueCreated { get; }
         public T Value { get; }
     }
+    public sealed class FastLazy<S, T>
```

Comment:
Why can't we push https://github.com/Aaronontheweb/FastAtomicLazy to nuget and use it that way?
BTW @Aaronontheweb I've created an issue and PR to FastAtomicLazy, please check it.